lke_clusters.go

```diff
@@ -35,7 +35,7 @@ type LKEClusterCreateOptions struct {
 	NodePools []LKEClusterPoolCreateOptions `json:"node_pools"`
 	Label     string                        `json:"label"`
 	Region    string                        `json:"region"`
-	Version   string                        `json:"version"`
+	Version   string                        `json:"k8s_version"`
```

Comment:
`Version` may be surprising to anyone using the Linode API docs as a reference.  I'm sure there are other patterned examples of the go field not matching the upstream name, but this one jumps out. It is understandable because of the change in the API, but I wonder if the field should follow the API here.